frontend/src/hooks/useGame.ts

```diff
@@ -15,15 +15,14 @@ export interface StateGameInit {
     enemyId: number;
 }
 
-// Initialize game state
-const initGame = (stateGameInit: StateGameInit) => usePost<StateGameInit, State>(`/combat/initialize` , stateGameInit);
+const initGame = (stateGameInit: StateGameInit) => 
+    usePost<StateGameInit, State>(`/combat/initialize` , stateGameInit);
+const useGameState = (fightId: number) => useGet<State>(`/combat/${fightId}`);
 
-// Combat action hook
-const useCombat = (fightId: number, action: GameAction) => {
-    return usePost<GameAction, State>(`/combat/${fightId}/action`, action);
+const useCombat = (fightId: number) => {
+    const performAction = (action: GameAction) =>
```

Comment:
Jeg er ikke helt sikker på hvorfor du wrapper det i en action du exporter, er det ikke muligt at holde syntaxen for hooksne på samme måde hvor det fx ville være

const useCombat = (fightId: number, action: GameAction) => usePost<GameAction, State>(`/combat/${fightId}/action`, action)

Reply:
hooken ser ud som de andre

Reply:
Nej den gør ikke